oauthdoctor/oauth/helper.go

```diff
@@ -264,13 +266,14 @@ func (c *Config) oauth2Client(code string) (*http.Client, string) {
 var apiURL = "https://googleads.googleapis.com/v1/customers/"
 
 // getAccount makes a HTTP request to Google Ads API customer account
-// endpoint and parse the JSON response.
+// endpoint and parses the JSON response.
 func (c *Config) getAccount(client *http.Client) (*bytes.Buffer, error) {
 	req, err := http.NewRequest("GET", apiURL+c.CustomerID, nil)
 	if err != nil {
 		return nil, err
 	}
 
+	req.Header.Set("User-Agent", userAgent())
```

Comment:
Put header filed names in lowercase, the HTTP2 standard. Some time next year we will be enforcing this.

Reply:
Done